src/views/credentials/addCredentialModal.tsx

```diff
@@ -32,13 +37,23 @@ interface CredentialFormType extends Partial<CredentialType> {
 
 interface CredentialFormFieldsProps {
   formData?: CredentialFormType;
+  errors?: CredentialErrorType;
   authType?: string;
   typeValue?: string;
   setAuthType?: (credentialType: string) => void;
   handleInputChange?: (field: string, value: string) => void;
 }
 
-const useCredentialForm = (credentialType: string | undefined, credential?: CredentialType) => {
+const USER_PASS = 'Username and Password';
+const TOKEN = 'Token';
+const SSH_KEY = 'SSH Key';
+
+const useCredentialForm = (
+  credentialType: string | undefined,
+  errors: CredentialErrorType,
+  setFormErrors: any,
```

Comment:
I couldn't find the exact type of this. This is the function that updates the `errors` state.